code/src/java/pcgen/io/PCGIOHandler.java

```diff
@@ -257,7 +257,7 @@ public void read(PlayerCharacter pcToBeRead, InputStream in,
 		}
 	}
 
-	private boolean isPCGCersion2(List<String> lines)
+	private static boolean isPCGCersion2(List<String> lines)
```

Comment:
Why should they be instance methods?  It acts a lot more like a helper method than a property of the object itself.  Perhaps I'm wrong?